internal/controller/feature/features.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2020 The cert-manager Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package feature
+
+import (
+	"k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/component-base/featuregate"
+
+	utilfeature "github.com/cert-manager/istio-csr/pkg/util/feature"
+)
+
+const (
+	// alpha: v0.7.2
+	//
+	// AdditionalAnnotations allows additional annotations to be specified on
+	// certificate requests
+	AdditionalAnnotations featuregate.Feature = "AdditionalAnnotations"
```

Comment:
Is it really necessary to make this feature gated? Seems like a no-brainer good feature to me. Are we concerned that the flag schema is going to change?

Reply:
Yep - fair enough. We came at it initially from a different angle, but there's no real need for it.